### DISCUSSION TITLE
Could you please guide me in the way to add cool background to the documentation like you added

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#1060] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/9841959c-891b-410d-9470-f836fbadf833)

I am also testing with radial gradients behind it. Thanks. 

**You can just tell me the path to the code file which does this, I will do the remaining things**

**Comment by fuma-nama** [comment]
https://magicui.design/docs/components/grid-pattern

**Comment by Gitstar-OC** [comment]
I know the library, I was asking for how did you add those things behind the mdx and path to code of your implementation in fumadocs. I want to add a gradient round circle of my choice behind it.

**Reply by olotfi** [reply, marked as answer]
Hey @Gitstar-OC, I was looking at doing something similar and liked how it was done in Fumadocs.

You should probably add the gradient + svg in your docs root `layout.ts`. So for example `app/docs/layout.ts`. The one that has `DocsLayout` component from Fumadocs. You will have to play around with the positioning to get it right for your case.

The one implemented in Fumadocs is the `span` here: https://github.com/fuma-nama/fumadocs/blob/67f860a5a7bf6b9a76ee7643c17133bafa86331d/apps/docs/app/docs/layout.tsx#L44

**Comment by Gitstar-OC** [comment]
Yes, thanks for the helping me. I got the component the same day when I checked in the developer tools.
